tide-core/src/app.rs

```diff
@@ -31,6 +31,7 @@ use crate::{
 ///
 /// ```rust, no_run
 /// #![feature(async_await)]
+/// # use tide_core as tide;
```

Comment:
The way `futures` deals with this "writing doc examples from the perspective of the facade" is to add `futures` as a dev-dependency of `futures-utils` etc. You could do the same here, add `tide` as a dev-dependency of `tide-core` and the examples will work to reference the types through the facade.

Reply:
Interesting. Will do.